899 TC- WalkerColina -03

wcolina opened the pull request.

wcolina committed:
Automatd test case for Todos los datos de equipos de protección personal son mostrados en una lista.

wcolina committed:
Delete unnecesary files.

wcolina commented:
**Test Case:**

![image](https://user-images.githubusercontent.com/852076/41892898-d7ee22ae-78e7-11e8-9288-29919eab9493.png)

**Automated Test Case Result:**

![image](https://user-images.githubusercontent.com/852076/41892880-c31f63a6-78e7-11e8-8fe8-6080f6d1af52.png)

airmiky commented on src/main/java/org/umssdiplo/automationv01/core/managepage/ppe/PPEList.java:
```diff
@@ -0,0 +1,26 @@
+    private WebElement ppeContainer;
+
+    public PPEList() {
+        CommonEvents.isPresent(ppeContainer);
```
usar CommonEvents.isVisible(ppeContainer);

wcolina replied:
Done.

airmiky commented on src/test/resources/features/ppe/PPEList.feature:
```diff
@@ -0,0 +1,9 @@
+
+  Scenario: Equipo de proteccion personal - Todos los datos de equipos de protección personal son mostrados en una lista
+    Given 'SSI-A' home page is loaded
+    And Click 'PPE' menu on 'Header' page
```
Corregir a minuscula "click"

wcolina replied:
Done.

airmiky commented on src/test/resources/features/ppe/PPEList.feature:
```diff
@@ -0,0 +1,9 @@
+  Scenario: Equipo de proteccion personal - Todos los datos de equipos de protección personal son mostrados en una lista
+    Given 'SSI-A' home page is loaded
+    And Click 'PPE' menu on 'Header' page
+    And Click 'PPE' sub menu of 'PPE' menu
```
Corregir a minuscula "click"

wcolina replied:
Done.

airmiky reviewed (commented):
Revisar las observaciones por favor.

wcolina committed:
First merge commit.

wcolina committed:
Merge remote-tracking branch 'remotes/origin/development' into TC-899

wcolina committed:
Final changes for automated test case:

EPP - Lista - Todos los datos de equipos de protección personal son mostrados en una lista después de seleccionar la opción "PPE" del menú "PPE"

wcolina committed:
Fix merge.

wcolina commented:
Fix all comments and merged.
